manifests/d/Dagger/Cli/0.9.6/Dagger.Cli.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Dagger.Cli
```

```suggestion
PackageIdentifier: Dagger.DaggerCli
```
- Please remember to update the file path and file name to match this change.